android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java

```diff
@@ -87,30 +89,33 @@ public void render () {
         }
         camController.update();
 
-        Gdx.gl.glViewport(0, 0, Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
+        if (carShouldBeRotated) {
+            Gdx.gl.glViewport(0, 0, Gdx.graphics.getHeight(), Gdx.graphics.getWidth());
+        } else {
+            Gdx.gl.glViewport(0, 0, Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
+        }
         Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT | GL20.GL_DEPTH_BUFFER_BIT);
 
         modelBatch.begin(cam);
         modelBatch.render(instances, environment);
         modelBatch.end();
     }
 
-    @Override
-    public void dispose () {
-        modelBatch.dispose();
-        instances.clear();
-        assets.dispose();
-    }
-
     @Override
     public void resize(int width, int height) {
-    }
+        Vector3 temporaryCamPosition = cam.position;
 
-    @Override
-    public void pause() {
-    }
+        if (carShouldBeRotated) {
+            cam = new PerspectiveCamera(60, height, width);
+        } else {
+            cam = new PerspectiveCamera(60, width, height);
+        }
```

Comment:
It should be in the factory class.

Reply:
Problem is no longer present after refactoring.